stdlib/private/StdlibCollectionUnittest/MinimalCollections.swift.gyb

```diff
@@ -701,10 +701,10 @@ public struct ${Self}<T> : ${SelfProtocols} {
 %     end
   }
 
-  public subscript(bounds: Range<${Index}>) -> ${SelfSlice}<${Self}<T>> {
+  public subscript(bounds: Range<${Index}>) -> Slice<${Self}<T>> {
```

Comment:
You can remove the definition of SelfSlice too right?

Reply:
It's still used within the gyb. These tests create `Minimal${SelfSlice}` opaque types for some tests.

I did miss some more `${SelfSlice}` uses tho.

Reply:
No wait that is a different test. These ones just reuse `SelfSlice` to create `MinimalCollection` names. But still, they need something like it.

Reply:
I felt guilty after this so have fixed it properly now...